### ISSUE TITLE
Package not compatible with React Native 0.56 (Android)

### ISSUE THREAD

**Issue by taranda** [react-native-community/react-native-svg#730]
When attempting to include this package  into a React Native 0.56 project, the following error occurs when running `react-native run-android`:

```
The SDK Build Tools revision (23.0.1) is too low for project ':react-native-svg'. Minimum required is 25.0.0
```

This is likely due to the fact that React Native 0.56 updates the buildTools version and targets API 26.  Targeting API 26 is [mandated](https://developer.android.com/distribute/best-practices/develop/target-sdk) for all new apps that wish to publish to Google Play.

In other words, unless this problem is fixed, new apps will not be able to use this package and publish to Google Play starting on 1 August.

Here is the relevant portions of the `package.json` for the app that generated this error:

```
"dependencies": {
    "react": "16.4.1",
    "react-native": "0.56.0",
    "react-native-config": "^0.11.5"
  }
```

I think [PR #729](https://github.com/react-native-community/react-native-svg/pull/729) may fix this issue.

**Comment by haixoay96** [comment]
in build.gradle at android folder 
subprojects { subproject ->
    afterEvaluate{
        if((subproject.plugins.hasPlugin('android') || subproject.plugins.hasPlugin('android-library'))) {
            android {
                compileSdkVersion rootProject.ext.compileSdkVersion
                buildToolsVersion rootProject.ext.buildToolsVersion
            }
        }
    }
}

**Comment by taranda** [comment]
@haixoay96 Nice workaround.  I still hope we we can get #729 merged so this works “out of the box.”

**Comment by RodolfoSilva** [comment]
Workaround:

```
subprojects { subproject ->
    afterEvaluate {
        if ((subproject.plugins.hasPlugin('android') || subproject.plugins.hasPlugin('android-library'))) {
            android {
                compileSdkVersion rootProject.ext.compileSdkVersion
                buildToolsVersion rootProject.ext.buildToolsVersion
            }
        }
    }
}
```

**Comment by RodolfoSilva** [comment]
Workaround, put it in build.gradle:

```
subprojects { subproject ->
    afterEvaluate {
        if ((subproject.plugins.hasPlugin('android') || subproject.plugins.hasPlugin('android-library'))) {
            android {
                compileSdkVersion rootProject.ext.compileSdkVersion
                buildToolsVersion rootProject.ext.buildToolsVersion
            }
        }
    }
}
```

**Comment by msand** [comment]
v6.5.0 and up resolved this. Will close this now.
